pkg/crypto/pseudo.go

```diff
@@ -58,7 +59,7 @@ func NodePseudo(nodes []t.NodeID, clients []t.ClientID, ownID t.NodeID, seed int
 	if c == nil {
 		if err == nil {
 			// Own ID was not found and Crypto module instantiation was not even attempted.
-			return nil, fmt.Errorf("ownID (%d) not found among nodes", ownID)
+			return nil, fmt.Errorf("ownID (%s) not found among nodes", ownID)
```

Comment:
Let's use `%v` so we don't need to change this again in the unlikely case we use a different type for IDs in the future (or in a variant of Mir)

Reply:
ok